sdk/txbuild/test_test.go

```diff
@@ -52,10 +52,10 @@ func Test(t *testing.T) {
 		seqNum, err := account.GetSequenceNumber()
 		require.NoError(t, err)
 		tx, err := txbuild.CreateEscrow(txbuild.CreateEscrowParams{
```

Comment:
the file name (`test_test`) and test name (`Test`) are a little confusing as they are fairly generic. Makes it hard to understand what the test is doing or what component it is testing without reading through the full code.

Not something to be fixed in this PR though as that would complicate it unnecessarily. Just an FYI for posterity, can fix in a separate PR at some point

Reply:
agreed, when we look at the testing as part of [this ticket](https://github.com/stellar/experimental-payment-channels/issues/70) probably we should rename this to something more descriptive (or possibly remove if this integration test doesn't test much more than `sdk/state/integration`)